### PR TITLE
Logging pde

### PR DESCRIPTION
**Pull request by aschroed**
1. Refactored main method of parse_hpoa.py to break it up a bit and make it easier to write tests.
2. Modified logging set up in all scripts that process Phenotype and Disease and Evidence items
3. Added tests for generate_items_from_owl.

### CODE REVIEW

File: src/encoded/tests/test_generate_item_from_owl.py [hunk 1]
```diff
@@ -0,0 +1,903 @@
...
+    obsterm = to_update[0]
+    assert obsterm['uuid'] == 'uuid10'
+    assert obsterm['status'] == 'obsolete'
+
+
+def test_id_post_and_patch_set_obsolete_false_do_not_obsolete_live_term(mocker, terms, mock_logger):
+    """ if set_obsolete is false then the extra dbterm should not be added to patches
+        as a term to set to obsolete as long as it's status is not obsolete or deleted
+    """
+    import copy
```
Comment (willronchetti) [inline src/encoded/tests/test_generate_item_from_owl.py]: We should get out of the habit of doing this, there is no real advantage in nearly all cases and it's generally considered "bad style"

File: src/encoded/commands/generate_items_from_owl.py [hunk 2]
```diff
@@ -303,7 +283,7 @@ def get_existing_items(connection, itype, include_obs_n_del=True):
     return terms
 
 
-def connect2server(env=None, key=None, keyfile=None):
+def connect2server(env=None, key=None, keyfile=None, logger=None):
```
[the remark this answers is not in the document]
Comment (aschroed) [reply src/encoded/commands/generate_items_from_owl.py]: I actually have no idea what do_the_thing is? So it's not me who is using that elsewhere.  I'm happy to use if it makes sense to do so - I just needed someway to to get a connection dict.

File: src/encoded/tests/test_generate_item_from_owl.py (hunk 1 above)
Comment (aschroed) [reply src/encoded/tests/test_generate_item_from_owl.py]: You mean do top level import instead I guess?

File: src/encoded/commands/generate_items_from_owl.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (willronchetti) [reply src/encoded/commands/generate_items_from_owl.py]: Sorry by `do_the_thing` I meant literally the use of underscores in all functions.

File: src/encoded/tests/test_generate_item_from_owl.py (hunk 1 above)
Comment (willronchetti) [reply src/encoded/tests/test_generate_item_from_owl.py]: Yeah just do a top level import

File: src/encoded/commands/generate_items_from_owl.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (aschroed) [reply src/encoded/commands/generate_items_from_owl.py]: Ah - bad habit I guess - I'm happy to change but frequently use -  to->2 and for->4 dropping the underscore as a matter of habit/preference - laziness about using the shift key. <shrug>

File: src/encoded/commands/load_items.py [hunk 3]
```diff
@@ -7,46 +7,85 @@
...
-date = datetime.now().strftime("%y-%m-%d-%H-%M-%S")
-logfile = 'load_items.log'
-logger = logging.getLogger(__name__)
-logging.config.dictConfig({
-    'version': 1,
-    'disable_existing_loggers': False,
-    'formatters': {
-        'standard': {
-            'format': '%(levelname)s:\t%(message)s'
+# '''logging setup
```
[the remark this answers is not in the document]
Comment (aschroed) [reply src/encoded/commands/load_items.py]: Done

File: src/encoded/commands/parse_hpoa.py [hunk 4]
```diff
@@ -143,8 +199,67 @@ def check_hpo_id_and_note_problems(fname, hpoid, hpoid2uuid, problems):
...
-def line2list(line):
-    return [d.strip() for d in line.split('\t')]
+def create_evi_annotation(data, hpoid2uuid, problems):
+    """ Looks at all the fields in the annotation line and
+        transforms to EvidenceDisPheno format
+    """
+    HP_REGEX = re.compile('^HP:[0-9]{7}')
+    pheno_annot = {}
+    for f, v in data.items():
+        if not v or (f == 'DiseaseName'):
```
[the remark this answers is not in the document]
Comment (aschroed) [reply src/encoded/commands/parse_hpoa.py]: Done

File: src/encoded/tests/test_generate_item_from_owl.py (hunk 1 above)
Comment (aschroed) [reply src/encoded/tests/test_generate_item_from_owl.py]: done

**Pull request merged**